BookingsApi/BookingsApi.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -172,7 +173,33 @@ public async Task SeedVideoHearing(VideoHearing videoHearing)
             
             _seededHearings.Add(videoHearing.Id);
         }
-        
+
+        public async Task CloneVideoHearing(Guid hearingId, IList<DateTime> datesOfHearing)
+        {
+            var dbContext = new BookingsDbContext(_dbContextOptions);
+            var hearing = await new GetHearingByIdQueryHandler(dbContext)
+                .Handle(new GetHearingByIdQuery(hearingId));
+            
+            var orderedDates = datesOfHearing.OrderBy(x => x).ToList();
+            var totalDays = orderedDates.Count + 1;
+            var commands = orderedDates.Select((newDate, index) =>
+            {
+                var hearingDay = index + 2; 
+                return CloneHearingToCommandMapper.CloneToCommand(hearing, newDate, new RandomGenerator(),
+                    GetSipAddress(hearing), totalDays, hearingDay);
```

Comment:
This can be any value; you don't need a function here to loop through all endpoints. Alternatively, the correct way would be to read it from the Configuration